mLRS/Common/hal/device_conf.h

```diff
@@ -381,4 +381,4 @@ The default selection of frequency bands can be overruled by feature defines.
 #define USE_FEATURE_FLRC
 #if defined TX_FRM303_F072CB || defined RX_FRM303_F072CB // is short of RAM for tx, and possibly too slow
   #undef USE_FEATURE_FLRC
-#endif
+#endif
```

Comment:
nit pick
could you please add the line so this file goes away from the changed files :)

Reply:
Huh - I copied these files directly from Main...

What's the difference?

Main:

![image](https://github.com/olliw42/mLRS/assets/41841496/674d6d60-2d0a-4075-a8c4-ea0f78ff4d23)

My Branch:

![image](https://github.com/olliw42/mLRS/assets/41841496/a0a46846-ccd2-468e-9224-4e3f739f1981)

Reply:
Resolved